docs/cronjobs.js

```diff
@@ -1,43 +1,35 @@
 import {html} from './zhtml.js';
 
-export function cronjobCDNStatusBadge() {
+function generateStatusBadge(workflowName) {
```

Comment:
This is nice! At this point I think you could've inlined `cronjobAutorollFirefox`, `cronjobAutorollWebKit` and others - buy I'm fine either way.